exchanges/okx/okx_types.go

```diff
@@ -620,13 +620,13 @@ type CancelOrderRequestParam struct {
 
 // AmendOrderRequestParams represents amend order requesting parameters.
 type AmendOrderRequestParams struct {
-	InstrumentID            string  `json:"instId"`
-	CancelOnFail            bool    `json:"cxlOnFail"`
-	OrderID                 string  `json:"ordId"`
-	ClientSuppliedOrderID   string  `json:"clOrdId"`
-	ClientSuppliedRequestID string  `json:"reqId"`
-	NewQuantity             float64 `json:"newSz,string"`
-	NewPrice                float64 `json:"newPx,string"`
+	InstrumentID            string  `json:"instId,omitempty"`
```

Comment:
Thanks for that! Just keep this one
```suggestion
	InstrumentID string `json:"instId"`
```